.golangci.yml

```diff
@@ -74,7 +99,11 @@ issues:
         - unused
     # Ignore "TLS MinVersion too low", "TLS InsecureSkipVerify set true", and "Use of weak random
     # number generator (math/rand instead of crypto/rand)" in tests.
-    - path: (.+)_test.go
+    - path: _test\.go
```

Comment:
Won't this regular expression now only match literally `_test.go` instead of all `_test` files?

Reply:
It depends on what matching rules `golangci-lint` is using, but generally the regex `_test\.go` should match any string containing the pattern "_test.go" because there is no `$` (the "beginning of string" anchor). I updated that path based on the `exclude-rules` example in the [`golangci-lint` config file docs](https://golangci-lint.run/usage/configuration/#config-file).